src/utils/dom.js

```diff
@@ -15,6 +15,12 @@ export const states = {
  * Add modal + overlay to DOM
  */
 export const init = (params) => {
+  // Clean up old modals
+  while (getContainer()) {
```

Comment:
@limonte Ok, if it is doing it everytime then the loop is not needed. I'll update it